lms/static/sass/views/_learner-profile.scss

```diff
@@ -307,8 +308,244 @@
             border: 2px dashed $link-color;
             .u-field-title,
             i {
-              color:  $link-color;
+                color: $link-color;
+            }
+        }
+    }
+
+    .badge-paging-header {
+        padding-top: $baseline;
+    }
+
+    .pagination {
```

Comment:
We already had pagination in the LMS but it was scoped too tightly to teams. @frrrances made it more available here:

https://github.com/edx/edx-platform/pull/10997